examples/dns/README.md

```diff
@@ -1,19 +1,48 @@
-# DNS
+# Domain Name Service (DNS)
 
-This example creats a domain `example.exo` with three records
+This example demonstrates how to manage
+[Domain Name Service (DNS)](https://community.exoscale.com/documentation/dns/)
+and point Resource Records (RR) to compute instances, using `exoscale_domain` and
+`exoscale_domain_record` resources.
 
-- **A** so that `example.exo` points to the IPv4 of the instance (`exoscale_compute.<name>.ip_address`)
-- **CNAME** to alias `www.example.exo` to `example.exo`
-- **TXT** to put a text field
+It creates the `example.exo` domain with four records:
+- the (bare) domain **A**, pointing to the instance IPv4
+- its **AAAA** sibling, pointing to the instance IPv6
+- a **TXT** record
+- a **CNAME**, aliasing `www.example.exo` to the (bare) domain
 
-```
+```console
 $ terraform init
+$ terraform apply \
+  -var exoscale_api_key=$EXOSCALE_API_KEY \
+  -var exoscale_api_secret=$EXOSCALE_API_SECRET
+
+...
 
-$ terraform apply
+Outputs:
+
+my_instance_ipv4 = "185.19.31.186"
+my_instance_ipv6 = "2a04:c43:e00:63a3:43b:eaff:fe00:3f6"
 ```
 
-Then, on the Portal, you should see the following entries.
+Which can then be seen using [Exoscale CLI](https://github.com/exoscale/cli/):
 
-![expected output in the portal](example-exo.png)
+```console
+$ exo dns show example.exo
```

Comment:
CLI output is not up-to-date.  Latest CLI (1.58.0) uses API v2 for DNS commands and IDs are now in UUID format.